ui/admin/tests/unit/controllers/scopes/scope/auth-methods/index-test.js

```diff
@@ -6,6 +6,15 @@
 import { module, test } from 'qunit';
 import { setupTest } from 'ember-qunit';
 import { waitUntil } from '@ember/test-helpers';
+import { visit } from '@ember/test-helpers';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import { setupIndexedDb } from 'api/test-support/helpers/indexed-db';
+import {
+  authenticateSession,
+  // These are left here intentionally for future reference.
+  //currentSession,
+  //invalidateSession,
```

Comment:
Do we need these? I don't think i've ever used them lol

Reply:
yeah we don't, I also haven't used them. I'll clean this up, good catch!